index.html

```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
         </ul>
         </div>
     </section>
+    <section class="work-section" id="work-section">
+        <div class="work">
+            <h1 class="grid1">My recent works<img src="./Normal_Button/Vector.png"></h1>
```

Comment:
- [ ]  Please use only one h1 tag in each of your document. This is best SEO practice. Kindly change this h1 tag to any other heading tag between h2 and h5